lib/post-processing/image.js

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
I would prefer not using `-` in directory name. some Operating Systems/tools might give a hard time.

Reply:
right...what about `postprocessing.js`?